src/openfermion/utils/_trotter_exp_to_qgates.py

```diff
@@ -253,6 +272,8 @@ def trotterize_exp_qubop_to_qasm(hamiltonian,
         term_ordering (list of (tuples of tuples)): list of tuples
             (QubitOperator terms dictionary keys) that specifies
             order of terms when trotterizing
+        qubit_list: a list of names for qubits (otherwise they will be labeled
```

Comment:
Same comment as above about more explicit type descrtiption

Reply:
Copied the description from the above string